mscore/pagesettings.cpp

```diff
@@ -130,6 +124,7 @@ void PageSettings::blockSignals(bool block)
       pageHeight->blockSignals(block);
       pageOffsetEntry->blockSignals(block);
       pageGroup->blockSignals(block);
+// ???Radio button signals not blocked???
```

Comment:
The current code never blocks signals for the radio buttons.  Is this by design, or should I add it?  I haven't seen any problems, but I haven't stress-tested the UI either.